### PR TITLE
[fix] Set upper version limitations for some critical libs

### PR DESCRIPTION
**Pull request by mihran113**

### CODE REVIEW

**Review by gorarakelyan — APPROVED** [review]
lgtm

File: aim/utils/deprecation.py [hunk 1]
```diff
@@ -5,6 +5,16 @@
 DEFAULT_MSG_TEMPLATE = 'This functionality will be removed in'
 
 
+def sqlalchemy_version_check():
+    import sqlalchemy
+    import packaging.version
+    from aim.__version__ import __version__
+    if packaging.version.parse(sqlalchemy.__version__) >= packaging.version.parse('2.0.0'):
+        raise RuntimeError(f'Aim v{__version__} does not support sqlalchemy v{sqlalchemy.__version__}. '
+                           f'Please check this is issue for further updates: '
```
Comment (alberttorosyan) [inline aim/utils/deprecation.py:14]: ```f'Please check the following issue for further updates```

**Review by alberttorosyan — APPROVED** [review]
Looks good! Left a small comment wrt error message.
Please update the CHANGELOG.md as well

**Comment by mihran113** [comment]
Resolves #2510

**Pull request merged**